lld/test/MachO/stabs-abs-path.s

```diff
@@ -0,0 +1,58 @@
+# REQUIRES: x86
```

Comment:
Is there a need to create a new test file for this use case? This seems like this could be placed in [`stabs.s`](https://github.com/llvm/llvm-project/blob/main/lld/test/MachO/stabs.s)

Reply:
I was hesitant to try to add another file there given it's complexity, but I'll give it a stab.